src/Npgsql/NpgsqlCommand.cs

```diff
@@ -893,21 +877,17 @@ async Task SendDeriveParameters(bool async)
             BeginSend();
             var connector = Connection.Connector;
             Debug.Assert(connector != null);
-            var buf = connector.WriteBuffer;
             for (var i = 0; i < _statements.Count; i++)
             {
                 async = ForceAsyncIfNecessary(async, i);
 
-                await connector.ParseMessage
-                    .Populate(_statements[i].SQL, string.Empty)
-                    .Write(buf, async);
+                var statement = _statements[i];
 
-                await connector.DescribeMessage
-                    .Populate(StatementOrPortal.Statement, string.Empty)
-                    .Write(buf, async);
+                await connector.WriteParse(statement.SQL, "", EmptyParameters, async);
+                await connector.WriteDescribe(StatementOrPortal.Statement, string.Empty, async);
```

Comment:
Maybe align everything (here and at least one of the `WriteBind` calls above) to use `string.Empty` for consistency?

Reply:
I think `""` and `String.Empty` are equivalent in every way. I've changed to `string.Empty` specifically when calling the Write*() methods to make it a bit clearer/harder to miss. However, the codebase contains lots of `""` elsewhere and I'm not sure it's worth it going over everything just for consistency etc.

Reply:
Oh, I definitely didn't mean to imply "everything, everywhere", just in this newly modified code.